src/Chainweb/Version.hs

```diff
@@ -565,26 +565,29 @@ window Development = Just $ WindowWidth 120
 window Testnet03 = Just $ WindowWidth 120
 window Mainnet01 = Just $ WindowWidth 120
 
-txSilenceEndDate :: ChainwebVersion -> Maybe (Time Micros)
-txSilenceEndDate Test{} = Nothing
-txSilenceEndDate TimedConsensus{} = Nothing
-txSilenceEndDate PowConsensus{} = Nothing
-txSilenceEndDate TimedCPM{} = Nothing
-txSilenceEndDate FastTimedCPM{} = Nothing
-txSilenceEndDate Development = Nothing
-txSilenceEndDate Testnet03 = Nothing
-txSilenceEndDate Mainnet01 = Just [timeMicrosQQ| 2019-12-05T00:00:00.0 |]
+-- | This is used in a core validation rule and has been present for several
+-- versions of the node software. Changing it risks a fork in the network.
+--
+txEnabledDate :: ChainwebVersion -> Maybe (Time Micros)
+txEnabledDate Test{} = Nothing
+txEnabledDate TimedConsensus{} = Nothing
+txEnabledDate PowConsensus{} = Nothing
+txEnabledDate TimedCPM{} = Nothing
+txEnabledDate FastTimedCPM{} = Nothing
+txEnabledDate Development = Nothing
+txEnabledDate Testnet03 = Nothing
+txEnabledDate Mainnet01 = Just [timeMicrosQQ| 2019-12-05T00:00:00.0 |]
 
 -- | KILLSWITCH: The date after which nodes in the 1.1.x series will
 -- spontaneously allow Transactions in the system. This constant can be removed
 -- once the date has passed, and /must not be used in core validation code/.
 --
-txActivationDate :: ChainwebVersion -> Maybe (Time Micros)
-txActivationDate Test{} = Nothing
-txActivationDate TimedConsensus{} = Nothing
-txActivationDate PowConsensus{} = Nothing
-txActivationDate TimedCPM{} = Nothing
-txActivationDate FastTimedCPM{} = Nothing
-txActivationDate Development = Nothing
-txActivationDate Testnet03 = Nothing
-txActivationDate Mainnet01 = Just [timeMicrosQQ| 2019-12-06T00:00:00.0 |]
+transferActivationDate :: ChainwebVersion -> Maybe (Time Micros)
+transferActivationDate Test{} = Nothing
+transferActivationDate TimedConsensus{} = Nothing
+transferActivationDate PowConsensus{} = Nothing
+transferActivationDate TimedCPM{} = Nothing
+transferActivationDate FastTimedCPM{} = Nothing
+transferActivationDate Development = Nothing
+transferActivationDate Testnet03 = Nothing
+transferActivationDate Mainnet01 = Just [timeMicrosQQ| 2019-12-06T00:00:00.0 |]
```

Comment:
It's our intention that this code will be removed almost immediately after the specified date. Note that this specific date here gives a 24 hour gap between the "shut off" of the old nodes.